parser/src/function.rs

```diff
@@ -349,7 +350,25 @@ impl<'input> InlinedFunction<'input> {
         Function::from_offset(hash, self.abstract_origin)
     }
 
-    /// The size of the inlined function.
+    /// The address of the inlined function instance.
+    #[inline]
+    pub fn address(&self) -> Option<u64> {
+        self.address.get()
+    }
+
+    /// The address ranges of the function.
+    pub fn range(&self) -> Option<Range> {
+        if let (Some(address), Some(size)) = (self.address(), self.size()) {
+            Some(Range {
+                begin: address,
+                end: address + size,
```

Comment:
This is incorrect for instances that have multiple ranges. We should do the same as we already do for `Function`: store `Vec<Range>` in the struct have a `ranges` method that returns `&[Range]`.

Reply:
Done